package/MDAnalysis/core/AtomGroup.py

```diff
@@ -671,6 +749,7 @@ def force(self, vals):
         except (AttributeError, NoDataError):
             raise NoDataError("Timestep does not contain forces")
 
+    @deprecate(message="{}; use `position` property instead".format(_FIFTEEN_DEPRECATION))
     def centroid(self):
```

Comment:
what? the `centroid` isn't the same as the `position` array in my understanding.